katello/katello-remove

```diff
@@ -103,13 +103,12 @@ RPMS=(
     `rpm -qa | grep ^python-isodate`
     `rpm -qa | grep gutterball`
```

Comment:
@beav gutterball is already being removed in katello-remove I have modified katello-backup which is coming in a new PR